include/wx/generic/listctrl.h

```diff
@@ -208,6 +208,8 @@ class WXDLLIMPEXP_CORE wxGenericListCtrl: public wxNavigationEnabled<wxListCtrlB
     wxListHeaderWindow  *m_headerWin;
     wxListMainWindow    *m_mainWin;
 
+    void SetListRulesAlternateColourOnBlank(const bool state, const wxColour& colour);
```

Comment:
What is the reason for having separate alternate colors for empty and non-empty rows?

If the color was the same, perhaps the method could be called something like (adhering to an "Enable...") naming pattern) `EnableRulesAndAlternateRowColoursForBlankRows(bool enable)`. Yes, it is horribly long but ATM I cannot think of anything shorter and matching the naming pattern.